Week2/homework/js exercises/3-RecipeCard.js

```diff
@@ -0,0 +1,16 @@
+//DECLARE A VARIABLE THAT HOLDS AN OBJECT (YOUR MEAL RECIPE)
+//GIVE THE OBJECT 3 PROPERTIES:
+//A TITLE(STRING), A SERVINGS(NUMBER), 
+//AND AN INGREDIENTS (ARRAY OF STRINGS) PROPERTY.
+
+const mySpecialRecipe = [{
+	mealname: "Vla",
+	serves :3,
+	ingredients: ["1 lt. milk", "200 gr. sugar", "1 banana", "20 gr. starch"]
+}]
```

Comment:
In this case you declared an array containing an object, but the exercise was about creating a "object" itself. 

Please note that te solution provided won't hold up anymore, can you redo this assignment using the following:

```
const mySpecialRecipe = {
	mealname: "Vla",
	serves :3,
	ingredients: ["1 lt. milk", "200 gr. sugar", "1 banana", "20 gr. starch"]
}
```
I'll give a little hint using `in` instead of `of`, please note to log each property out separately as described in the exercise.